Mathlib/Data/PFunctor/Univariate/Basic.lean

```diff
@@ -24,26 +24,26 @@ An element of `P α` is a pair `⟨a, f⟩`, where `a` is an element of a type `
 `f : B a → α`. Think of `a` as the shape of the object and `f` as an index to the relevant
 elements of `α`.
 -/
-@[pp_with_univ]
+@[pp_with_univ, nolint checkUnivs]
```

Comment:
I can believe that the nolint here is justified, but it might help to add a comment why the lint is a false-positive